### PR TITLE
move integration technologies blog to docs

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/node-red/integration-technologies/rest.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+## Example: Reading Data
+
+Now that you've learned how to create REST APIs in Node-RED, let's explore an example of reading data using a HTTP GET request. This example will demonstrate how to fetch data from an external API and process it and display on dashboard chart.
+
+For the example we will fetch the data of Node-RED Dashboard 2.0 Downloads from npm registry api.
+`https://api.npmjs.org/downloads/range/last-month/@flowforge/node-red-dashboard`.
+
+A simple flow to fetch data from npm registry  would be:
+
+<iframe width="100%" height="225px" src="https://flows.nodered.org/flow/7c2dd3ccde70746a40ef8f5aa58c591c/share?height=100" allow="clipboard-read; clipboard-write" style="border: none;"></iframe>
```
Comment (ZJvandeWeg) [inline src/node-red/integration-technologies/rest.md:58]: Do we want to copy this one and put it into `renderFlow` elements?

File: src/node-red/integration-technologies/rest.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+but the chart would now have multiple data points
+for the same day. This paints multiple lines on top of each other. While that works,
+the hover of the chart will display the duplication and it's wastefull.
+
+So before we update the chart we need to send a message to the chart where the
+[payload is `[]`](https://dashboard.flowfuse.com/nodes/widgets/ui-chart.html#removing-data).
+That way the chart is emptied first, and right afterwards it will
+receive the new data to write.
+
+<iframe width="100%" height="500px" src="https://flows.nodered.org/flow/47f4cda247f2f2e0172ab61c795308bb/share" allow="clipboard-read; clipboard-write" style="border: none;"></iframe>
```
Comment (ZJvandeWeg) [inline src/node-red/integration-technologies/rest.md:124]: Let's put this one too in `renderFlow`

File: src/node-red/integration-technologies/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+eleventyNavigation:
+  key: "Integration Technologies"
+meta:
+   title: Using Different Technologies for Building Applications with Node-RED.
+   description: Learn how to leverage various integration technologies with Node-RED for building robust and interconnected applications.
+   keywords: node-red, flowfuse, node-red integration, integration technologies, webhook, rest api
+---
+
+# Integration Technologies
```
Comment (ZJvandeWeg) [inline src/node-red/integration-technologies/index.md:10]: What else is going to be in this category?

Comment (sumitshinde-84) [reply src/node-red/integration-technologies/index.md:10]: we need write content on GraphQL , ETL

Comment (sumitshinde-84) [reply src/node-red/integration-technologies/index.md:10]: If we name the category "Data Integration," then we can also include a Kafka blog here? but we will loose an opportunity to create one more category called "streaming platforms"

**Pull request merged**